.cloud-build/notebook-execution-test-cloudbuild-single.yaml

```diff
@@ -10,29 +10,29 @@ steps:
     entrypoint: /bin/sh
     args:
     - -c
-    - python3 .cloud-build/CheckPythonVersion.py -q
+    - ${_PYTHON_VERSION} .cloud-build/CheckPythonVersion.py -q
   # Create a virtual environment
   - name: ${_PYTHON_IMAGE}
     entrypoint: /bin/sh
     args:
     - -c
-    - python3 -m venv workspace/env
+    - ${_PYTHON_VERSION} -m venv workspace/env
   # Install Python dependencies
   - name: ${_PYTHON_IMAGE}
     entrypoint: /bin/sh
     args:
     - -c
     - . workspace/env/bin/activate &&
-      python3 -m pip -q install -U pip &&
-      python3 -m pip -q install -U -r .cloud-build/requirements.txt
+      ${_PYTHON_VERSION} -m pip -q install -U pip &&
```

Comment:
I actually don't think you need this part after calling `workspace/env/bin/activate`.

Try using `python -m pip -q install ...`

Reply:
The `python` used should be inferred from the env.

Reply:
Done: a70b32277fa885e60f3850b8ad294ba763798106